### DISCUSSION TITLE
I think we should use html / js / css for the game, what about you?

### DISCUSSION THREAD

**Discussion by shadowleafy** [shadowleafy/collab-game#4] [General]

**Comment by shadowleafy** [comment]
(also we should probably come up with a game idea)

**Comment by ackledotdev** [comment]
Couple of things

1. Move this to Discussions. If you can't find that, give me full access and I'll find it for you.
2. Yes, HTML5 and JS sounds like a good bet so that it's easy to run (no downloading, no compilation).
3. Yes, we should get a game idea.
